cbv3_django_prototype/cbv3_django_prototype/templates/base.html

```diff
@@ -83,7 +83,22 @@
       {% endif %}
 
       {% block content %}
-        <p>Use this document as a way to quick start any new project.</p>
+      <h1>Welcome to the codebuddies django-concept project</h1>
+      <p> Please find some useful commands below to help you get started on Django development</p>
+      <h2>Application urls</h2>
+      <ul>
+        <li><a href="http://127.0.0.1:8000/api/v1/resources/">http://127.0.0.1:8000/api/v1/resources/ </a> - browse the API for resources</li>
+        <li><a href="http://127.0.0.1:8000/admin/">http://127.0.0.1:8000/admin/ </a> - login using superuser credentials</li>
+        <li><a href="http://127.0.0.1:8000/auth/obtain_token/">http://127.0.0.1:8000/auth/obtain_token/admin/</a> - get a token</li>
+        <li><a href="http://127.0.0.1:8000/auth/users/">http://127.0.0.1:8000/auth/users/</a> - create a user</li>
+      </ul>
+      <h2>docker-compose commands</h2>
+      <ul>
+        <li><code>docker-compose run --rm manage</code> createsuperuser - How to create a superuser</li>
```

Comment:
May be `createsuperuser` should be inside `<code/>` tags to be consistent with other lines.

Reply:
that's a good catch, thanks @sunu  :)